transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__event_deduped.sql

```diff
@@ -0,0 +1,319 @@
+        , context_source_id
+        , context_source_type
+        , context_terminators_lastindex
+        , context_timezone
+        , context_traits_active_address_id
+        , context_traits_address
+        , context_traits_auth_provider
+        , context_traits_cellphone
+        , context_traits_city
+        , context_traits_date_of_birth
+        , context_traits_district
+        , context_traits_edition
+        , context_traits_email
+        , context_traits_event_tenant
+        , context_traits_firstname
+        , context_traits_id
+        , context_traits_identity
+        , context_traits_instance_id
+        , context_traits_item_in_cart
+        , context_traits_language
+        , context_traits_last_audience_visit
+        , context_traits_lastname
+        , context_traits_latitude
+        , context_traits_longitude
+        , context_traits_name
+        , context_traits_nb_of_audience_visit
+        , context_traits_order_id
+        , context_traits_org_id
+        , context_traits_phone
+        , context_traits_phone_number
+        , context_traits_portal_customer_id
+        , context_traits_proto_node_options
+        , context_traits_province
+        , context_traits_referrer
+        , context_traits_session_id
+        , context_traits_shipping_method_kind
+        , context_traits_subdistrict
+        , context_traits_tenant
+        , context_traits_use_oauth
+        , context_traits_user_id
+        , context_traits_version
+        , context_traits_version_cli
+        , context_traits_where
+        , context_user_agent
+        , context_user_agent_data_brands
+        , context_user_agent_data_mobile
+        , context_user_agent_data_platform
+        , context_useragent
+        , context_where
+        , count
+        , cta
+        , customized_name
+        , customized_visibility
+        , download
+        , duration
+        , email
+        , error
+        , errors
+        , field
+        , filter
+        , first
+        , first_effectiveness
+        , first_recomputations
+        , fresh
+        , from_page
+        , gfyid
+        , github
+        , gitlab
+        , group_constrained
+        , inapp_notice
+        , include_deleted
+        , inquiry
+        , inquiry_issue
+        , installed_version
+        , invite_count
+        , is_first_preload
+        , jira
+        , join_call
+        , keyword
+        , license_id
+        , longest_api_resource
+        , longest_api_resource_duration
+        , max_api_resource_size
+        , method
+        , metric
+        , mkt_tok
+        , my_auth
+        , name
+        , num_high
+        , num_invitations
+        , num_invitations_sent
+        , num_low
+        , num_medium
+        , num_of_request
+        , num_total
+        , org_id
+        , original_timestamp
+        , persistent_notifications
+        , playbooks
+        , plugin_id
+        , post_id
+        , privacy
+        , q
```

Comment:
might be worth removing the spurious columns before running the process.

Reply:
We should check values on these columns first. Let's do it in a separate PR in order to keep things simpler.